### ISSUE TITLE
Setting player.model removes all weapons

### ISSUE THREAD

**Issue by savvin-dev** [altmp/altv-issues#1290]
### Description of the problem

When setting player.model, the player loses all weapons and camera returns behind the player. Maybe, it just spawns a player in the same place, but with `player.spawn(...)` this does not happen
https://youtu.be/NFrsi1DCNcI

I used custom chat and commands for ease.
```
alt.onClient('cmd:skin', (player, skinName) => {
    player.model = alt.hash(skinName)
})
alt.onClient('cmd:respawn', (player) => {
    player.spawn(new alt.Vector3(-7.8197, 5.6967, 71.1347), 0)
})
```

### Reproduction steps

Give yourself a weapon and set player.model on any skin

### Expected behaviour

No weapon reset

### Operating system

Windows 11 21H2

### Version

release/8.6

**Comment by FabianTerhorst** [comment]
Spawn only sets a positon while model in gta isn't changeable without recreating the ped.

**Comment by FabianTerhorst** [comment]
Fixed in dev/15.0-dev332
